app/src/main/res/layout/fragment_events.xml

```diff
@@ -11,4 +11,15 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         />
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        ><fragment
```

Comment:
Define XML subcomponents on their own new lines.
android:layout_height="50dp"
        >
  <fragment (also tab this in)